libsolidity/formal/Predicate.cpp

```diff
@@ -183,7 +183,8 @@ string Predicate::formatSummaryCall(vector<smtutil::Expression> const& _args) co
 		fun->isFallback() ? "fallback" :
 		fun->isReceive() ? "receive" :
 		fun->name();
-	return fName + "(" + boost::algorithm::join(functionArgs, ", ") + ")";
+	solAssert(fun->annotation().contract, "");
```

Comment:
I guess, no free function can end up here, since it's always external calls?

Reply:
So far, yes. That will change soon, but free functions will need some refactoring anyway, would do that then